PowerEditor/src/Utf8_16.cpp

```diff
@@ -285,52 +284,64 @@ Utf8_16_Write::~Utf8_16_Write()
 	fclose();
 }
 
-FILE * Utf8_16_Write::fopen(const TCHAR *_name, const TCHAR *_type)
+bool Utf8_16_Write::fopen(const TCHAR *name)
 {
-	m_pFile = ::generic_fopen(_name, _type);
+	m_pFile.reset(new CFile(name, CFile::WRITE));
```

Comment:
`std::make_unique` is preferred over `new`

Reply:
Perhaps but there's nothing wrong with this code, correct me if I'm wrong.